src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -511,6 +516,25 @@ private String generateAnnotated(String descTemp, SyntaxElementInfo<?> info, @Nu
 //		desc = desc.replace("${element.by-addon}", addon);
 		desc = handleIf(desc, "${if by-addon}", false);
 
+		// Accepted Changers
+		// TODO grab the accepted data type as well (find a way to include that in annotation as well)
+		if (info instanceof ExpressionInfo<?,?>) {
+			List<Changer.ChangeMode> acceptedChangeModes = new ArrayList<>(4); // 4 is common, SET/ADD/REMOVE/DELETE
+			AcceptedChangeModes annotation = c.getAnnotation(AcceptedChangeModes.class);
+			if (annotation != null) {
+				acceptedChangeModes = Arrays.stream(annotation.value()).toList();
+			} else {
+				try {
+					acceptedChangeModes = ((SimpleExpression<?>) info.getElementClass().newInstance()).getAcceptedChangeModes().keySet().stream().toList();
```

Comment:
i am a little weary of this given that expressions really should be initialized before having anything done on them - this may very well lead to inaccurate results